x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/utils.ts

```diff
@@ -73,3 +73,14 @@ export const calculateName = ({
     return 'untitled';
   }
 };
+
+/*
+  Alert throttle has to be null in case there are no actions or actions are exectured per rule exectution
+*/
+export const getAlertThrottle = (throttle: string | null | undefined): null | string => {
```

Comment:
It looks like the only possible values for `throttle` to be are `'no_actions' | 'rule' | '1h' | '1d' | '7d'`. Is it possible to add a `throttle` type, remove the `string` and then type the parameter of this function to this new throttle type plus `null | undefined`?